packages/connectors/connector-qq/src/constant.ts

```diff
@@ -0,0 +1,52 @@
+import type { ConnectorMetadata } from '@logto/connector-kit';
+import { ConnectorPlatform, ConnectorConfigFormItemType } from '@logto/connector-kit';
+
+export const authorizationEndpoint = 'https://graph.qq.com/oauth2.0/authorize';
+export const accessTokenEndpoint = 'https://graph.qq.com/oauth2.0/token';
+export const userInfoEndpoint = 'https://graph.qq.com/user/get_user_info';
+export const openIdEndpoint = 'https://graph.qq.com/oauth2.0/me';
+
+export const defaultScope = 'get_user_info,get_unionId';
```

Comment:
it is better to leave comments showing these endpoint information so that maintainer and review can easily find corresponding reference.